LJAPI.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+import os
+import requests
+import transaction
+
+
+from LearningJournal.models import Entry
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import scoped_session, sessionmaker
+from zope.sqlalchemy import ZopeTransactionExtension
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+
+
+database_url = os.environ.get('DATABASE_URL', None)
+engine = engine_from_config({'sqlalchemy.url': database_url})
+DBSession.configure(bind=engine)
+
+json_entries = {}
+
+
+def get_json():
+    """Get and store journal json data."""
+    resp = requests.get('https://sea401d2.crisewing.com/api/export?apikey=' + os.environ.get('API_KEY'))
```

Comment:
It might be wise (and would certainly be safer) to check to be sure you got a value for `API_KEY` from the environment before you proceed.  Without that value, there's no point in calling the API, so it would be a good place to "fail early"